bsp/boards/MSP430F5438A/bsp.c

```diff
@@ -0,0 +1,809 @@
+//*****************************************************************************
+//! @file       bsp.c
+//! @brief      Board support package for MSP430F5438a on SmartRF06EB.
+//!
+//! Revised     $Date: 2013-09-18 13:18:48 +0200 (on, 18 sep 2013) $
+//! Revision    $Revision: 10587 $
```

Comment:
why do we keep this file in our BSP? Our BSP most probably reimplements all of that, no?